build.fsx

```diff
@@ -1,305 +1,140 @@
-// include Fake lib
-#r @"packages/build/FAKE/tools/FakeLib.dll"
+#r "paket: groupref build //"
+#load ".fake/build.fsx/intellisense.fsx"
```

Comment:
this part _shouldn't_ be necessary anymore for FSAC-based editors, right @matthid?

Reply:
Well, someone may want to edit it in non-FSAC-based editor (crazy, I know)

Reply:
As long as it doesn't work in VS 2019 I'd continue suggesting adding it, as it is still better than nothing.
Within the .fsx we have a `#if FAKE` in order to branch away IDEs which already support `#r "paket:`

Reply:
Yeah, I'd leave it for sake of VS and Rider.